backend/db/sql-generic.c

```diff
@@ -1109,7 +1109,7 @@ _backend_schema_get(gpointer backend_data, gpointer _batch, gchar const* name, b
 	return TRUE;
 
 _error:
-	if (G_UNLIKELY(!j_sql_reset(thread_variables->sql_backend, prepared->stmt, NULL)))
+	if (G_UNLIKELY(!thread_variables || !prepared || !j_sql_reset(thread_variables->sql_backend, prepared->stmt, NULL)))
```

Comment:
`thread_variables` and/or `prepared` may be `NULL` if initialization failed. Therefore, we must check it in error handling